bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -281,6 +281,24 @@ export default defineComponent({
 						],
 					},
 				},
+				{
+					name: "Hull",
+					callback: (canvas: HTMLCanvasElement, bezier: WasmBezierInstance, options: Record<string, number>): void => {
+						const hullPoints: Point[][] = JSON.parse(bezier.hull(options.t));
+						hullPoints.forEach((iteration: Point[]) => {
+							iteration.forEach((point: Point, index) => {
```

Comment:
Please color code the output, I currently just see a lot of red lines and dots but don't know how that translates to the function's output. Based on how I think this works, you can probably try coloring each of these iterations differently the same way that the reduce function picks its colors. Consider also making the connecting lines a darker/lighter or more desaturated color to avoid distracting from the actual output of the function, which is the circles for the points.